### DISCUSSION TITLE
10/24 김지균 블로그 생성

### DISCUSSION THREAD

**Discussion by kimjigyun** [Orm-camp/blog-feedback#28] [동글동글 3기 피드백]
[블로그는 아직 없는데 만들어보겠습니다.](url)

저는 깃허브 페이지로 블로그를 생성해보겠습니다.
앞으로 배운 내용을 정리하는 TIL 형식으로 블로그를 운영하려고 해요.
매주 시간을 내서 내 블로그를 업데이트하고 꾸며보겠습니다.

**Comment by Orm-camp** [comment]
깃허브 페이지 도전하신다니, 멋있습니다!
배운 내용을 꾸준히 정리하시면 4개월의 발자취가 가득 쌓일 것 같아요.
꾸준한 기록들이 지균님에게 큰 도움이 되어줄 것 같습니다!
활동 응원하겠습니다 😊

**Reply by kimjigyun** [reply]
일기 쓰기 지금까지는 오래가지 못했지만 .. ㅎㅎ 다시 한번 시도해보겠습니다.

**Comment by ghost** [comment]
지윤환: 블로그 생성하시느라 고생 많으셨습니다. 
앞으로도 화이팅입니다!
